Scheduled Github Actions updates have been failing for 5 months

The last successful pipeline was on [May 5th](https://github.com/niess/python-appimage/actions/runs/8955272597). All presumably with the same error:

```python3
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/pwd/python_appimage/__main__.py", line 125, in <module>
    main()
    ~~~~^^
  File "/pwd/python_appimage/__main__.py", line 112, in main
    command = import_module(module, package=__package__)
  File "/opt/python/cp313-cp313/lib/python3.13/importlib/__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1022, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/pwd/python_appimage/commands/build/manylinux.py", line 7, in <module>
    from ...appimage import build_appimage, relocate_python
  File "/pwd/python_appimage/appimage/__init__.py", line 1, in <module>
    from .build import build_appimage
  File "/pwd/python_appimage/appimage/build.py", line 8, in <module>
    from ..utils.deps import APPIMAGETOOL, ensure_appimagetool
  File "/pwd/python_appimage/utils/deps.py", line 5, in <module>
    from .fs import copy_file, copy_tree, make_tree
  File "/pwd/python_appimage/utils/fs.py", line 1, in <module>
    from distutils.dir_util import mkpath as _mkpath, remove_tree as _remove_tree
ModuleNotFoundError: No module named 'distutils'
Error: Process completed with exit code 1.
```

Personally, I did look into this because I'd like to use a Python AppImage with the new 3.13 release. The 3.13 AppImage also shows as being from May and there are only the source tarballs in there.

Note that it is fairly easy to build the AppImages onself via:

```bash
python3 -m pip install --force-reinstall git+https://github.com/niess/python-appimage.git@master
python3 -c 'import python_appimage.commands.build.manylinux as manylinux; manylinux.execute("_2_28_x86_64", "cp311-cp311")'
ls -la 'python3.11.10-cp311-cp311-manylinux_2_28_x86_64.AppImage'
```

Or with my fix:

```bash
python3 -m pip install --force-reinstall git+https://github.com/mxmlnkn/python-appimage.git@master
python3 -c 'import python_appimage.commands.build.manylinux as manylinux; manylinux.execute("_2_28_x86_64", "cp313-cp313")'
ls -la 'python3.13.0-cp313-cp313-manylinux_2_28_x86_64.AppImage'
```

--- a/python_appimage/utils/fs.py
+++ b/python_appimage/utils/fs.py
@@ -1,7 +1,6 @@
-from distutils.dir_util import mkpath as _mkpath, remove_tree as _remove_tree
-from distutils.file_util import copy_file as _copy_file
 import errno
 import os
+import shutil
 
 from .log import debug
 
@@ -14,7 +13,7 @@ def make_tree(path):
     '''Create directories recursively if they don't exist
     '''
     debug('MKDIR', path)
-    return _mkpath(path)
+    return os.makedirs(path, exist_ok=True)
 
 
 def copy_file(source, destination, update=False, verbose=True):
@@ -23,7 +22,14 @@ def copy_file(source, destination, update=False, verbose=True):
     name = os.path.basename(source)
     if verbose:
         debug('COPY', '%s from %s', name, os.path.dirname(source))
-    _copy_file(source, destination, update=update)
+    if os.path.exists(source) and (
+        not update
+        or (
+            not os.path.exists(destination)
+            or (os.path.getmtime(source) > os.path.getmtime(destination))
+        )
+    ):
+        shutil.copy(source, destination)
 
 
 def copy_tree(source, destination):
@@ -38,7 +44,7 @@ def copy_tree(source, destination):
     for root, _, files in os.walk(source):
         relpath = os.path.relpath(root, source)
         dirname = os.path.join(destination, relpath)
-        _mkpath(dirname)
+        os.makedirs(dirname, exist_ok=True)
         for file_ in files:
             src = os.path.join(root, file_)
             dst = os.path.join(dirname, file_)
@@ -70,6 +76,6 @@ def remove_tree(path):
     name = os.path.basename(path)
     debug('REMOVE', '%s from %s', name, os.path.dirname(path))
     try:
-        _remove_tree(path)
+        shutil.rmtree(path)
     except OSError:
         pass

